src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,12 +8,21 @@
  */
 public class Address {
 
+    public static final String DIVIDER = ", ";
     public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
-    public static final String ADDRESS_VALIDATION_REGEX = ".+";
-
-    public final String value;
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses has to be in format BLOCK, STREET, UNIT, POSTAL_CODE";
+    public static final String ADDRESS_VALIDATION_REGEX = "^.+$";
+    public static final int BLOCK_INDEX = 0;
```

Comment:
it is good that you are using constants to eliminate magic numbers :+1:

however, please check how you should name the variables? please recollect the discussion on this during the tutorial